js/validation.js

```diff
@@ -0,0 +1,10 @@
+const validation = {
+    isPhoneNumber:function(str) {
+        const pattern = /^\d+$/;
```

Comment:
We can use a better regex to check phone numbers. 
1. We can trim white spaces
2. Include country codes followed by hyphen
refer https://stackoverflow.com/questions/16699007/regular-expression-to-match-standard-10-digit-phone-number